webapp/src/dogsList/listWithDogs.tsx

```diff
@@ -41,11 +93,53 @@ const useStyles = makeStyles((theme: Theme) =>
       display: "flex",
       alignItems: "center",
       marginTop: "2vh",
+      flexGrow: 1,
+      alignSelf: "center",
     },
   })
 );
 
+scheme.configureHeader((builder) => {
+  builder
+    .registerConfig("md", {
+      position: "absolute",
+    })
+    .registerConfig("xs", {
+      position: "relative", // won't stick to top when scroll down
+    });
+});
+
+scheme.configureEdgeSidebar((builder) => {
+  builder
+    .create("unique_id", { anchor: "left" })
+    .registerTemporaryConfig("xs", {
+      width: 200, // 'auto' is only valid for temporary variant
+    })
+    .registerPermanentConfig("md", {
+      width: "15%", // px, (%, rem, em is compatible)
+      collapsible: true,
+      collapsedWidth: "6%",
+    });
+});
+//scheme.configureInsetSidebar
+
 export default function ListWithDogs() {
+  const state = store.getState() as State;
+  const [collapsed, setCollapsed] = useState(false);
+  const dogs = useSelector(
+    (state: State) => state.dogs as ILostDogWithPicture[]
+  );
+  const loading = useSelector((state: State) => state.loading);
+  const refreshRequired = useSelector(
+    (state: State) => state.dogsRequireRefresh
+  );
+  const lastPage = useSelector((state: State) => state.dogsLastPage);
+  const [filters, setFilters] = useState({
+    page: config.defaultFilters.page,
+    size: config.defaultFilters.size,
+  });
+  // is this page last?
+  //const lastPage = useSelector((state) => state.carsLastPage);
   const history = useHistory();
   const [cookies, setCookie, removeCookie] = useCookies();
```

Comment:
And cookies too. Is not used, you can remove it.

Reply:
The date is there since this component does not provide proper date by default, i changed it to 2012 so user has to click the date as a temporary solution